src/basic_recipes/text.jl

```diff
@@ -162,7 +162,7 @@ function texelems_and_glyph_collection(str::LaTeXString, fontscale_px, halign, v
     scales_2d = [Vec2f(x[3] * Vec2f(fs)) for x in els]
 
     texchars = [x[1] for x in els]
-    chars = [texchar.char for texchar in texchars]
+    glyphindices = [glyph_index(texchar) for texchar in texchars]
```

Comment:
```suggestion
    glyphindices = [MathTeXEngine.glyph_index(texchar) for texchar in texchars]
```

Reply:
Should this just be `FreeTypeAbstraction.glyph_index` from the PR, which you can extend in MathTeXEngine?